boards/xtensa/intel_adsp_cavs25/board.cmake

```diff
@@ -3,4 +3,8 @@
 board_set_flasher_ifnset(misc-flasher)
 board_finalize_runner_args(misc-flasher)
 
-board_set_rimage_target(tgl)
+if (CONFIG_IPC_MAJOR_3)
```

Comment:
Problem with this approach is that CONFIG_IPC_MAJOR_3 is not a Zephyr Kconfig. We can't have this on Zephy rside.